java/src/main/kotlin/org/jetbrains/research/testspark/java/JavaPsiHelper.kt

```diff
@@ -26,6 +26,9 @@ class JavaPsiHelper(private val psiFile: PsiFile) : PsiHelper {
 
     override val language: SupportedLanguage get() = SupportedLanguage.Java
 
+    override fun availableForGeneration(e: AnActionEvent): Boolean =
+        getCurrentListOfCodeTypes(e).map { it.first }.toList().contains(CodeType.CLASS)
```

Comment:
Why check only for classes, while in Kotlin, check is for any codeType?

Reply:
I suspect it should be `isNotEmpty()` instead of `contains`, shouldn't it?

Reply:
@arksap2002 Please, leave a comment within the implementation so that it becomes clear from code.

Reply:
Put a comment before this method.